recipes/check-internet-speed/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "check-internet-speed" %}
```

Comment:
Please use grayskull to generate Python recipes. Grayskull make it easier for us to review.

See https://github.com/conda-forge/staged-recipes?tab=readme-ov-file#grayskull---recipe-generator-for-python-packages-on-pypi for more information.

Reply:
@ocefpaf i have regenerated Python recipes useing grayskull, Please review